index.html

```diff
@@ -0,0 +1,521 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="author" content="Vrushabh Dhatrak" />
+    <meta name="author" content="Vrushabh Dhatrak" />
+    <link rel="stylesheet" href="./Design-System/styles/main.css" />
+    <link rel="stylesheet" href="./index.css" />
+    <script
+      src="https://memorable-ui.netlify.app/script/header.js"
+      defer
+    ></script>
+    <script
+      src="https://memorable-ui.netlify.app/script/navigation.js"
+      defer
+    ></script>
+    <script src="https://code.iconify.design/2/2.1.1/iconify.min.js"></script>
```

Comment:
why is this script not at the end of body? it does not have defer too

Reply:
Ok Done